crates/libcontainer/src/syscall/linux.rs

```diff
@@ -100,7 +100,7 @@ impl FromStr for MountAttrOption {
                 MOUNT_ATTR_RELATIME,
             )),
             "rnoatime" => Ok(MountAttrOption::MountAttrNoatime(false, MOUNT_ATTR_NOATIME)),
-            "ratime" => Ok(MountAttrOption::MountAttrNoatime(true, MOUNT_ATTR_NOATIME)),
+            "ratime" => Ok(MountAttrOption::MountAttrNoatime(true, MOUNT_ATTR__ATIME)),
```

Comment:
@YJDoc2 this `ratime` should use `MOUNT_ATTR__ATIME` ?

Reply:
Yes, it should be that one! :)

Reply:
Also, the MountAttrOption should also use MountAttrAtime, or something similar? not only the passed value.

Reply:
Yes, done changed.